tests/code/test_search.py

```diff
@@ -197,4 +201,21 @@ def test_getDepartmentsForSupervisors():
         assert supervisorDept.DEPT_NAME in departments
         assert nonSupervisorDept.DEPT_NAME in departments
 
+        supervisorDeptTable = SupervisorDepartment.create(supervisor=supervisorUser.supervisor.ID,
+                                                    department=supervisorTableTestDept)
+
+        # check to make sure we get both the departments that a user interacted with and departments in the tavle
+        departments = list(getDepartmentsForSupervisor(supervisorUser))

```

Comment:
This list casting and the one above is now kind of unnecessary because of the list cast you do in your function.

Reply:
We add the user to a new department so we have to recall the function as the old call was prior to the add and won't include the new department.